docs/kafka/manage-cluster/README.adoc

```diff
@@ -0,0 +1,250 @@
+
+//OpenShift Connectors
+:connectors: Connectors
+:product-long-connectors: OpenShift Connectors
+:product-connectors: Connectors
+:product-version-connectors: 1
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+:getting-started-url-connectors: connectors/getting-started-connectors/README.adoc
+:getting-started-rhoas-cli-url-connectors: connectors/rhoas-cli-getting-started-connectors/README.adoc
+:addon-url-connectors: https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01
+
+
+//OpenShift API Designer
+:product-long-api-designer: OpenShift API Designer
+:product-api-designer: API Designer
+:product-version-api-designer: 1
+:service-url-api-designer: https://console.redhat.com/application-services/api-designer/
+:getting-started-url-api-designer: api-designer/getting-started-api-designer/README.adoc
+
+//OpenShift API Management
+:product-long-api-management: OpenShift API Management
+:product-api-management: API Management
+:product-version-api-management: 1
+:service-url-api-management: https://console.redhat.com/application-services/api-management/
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-cluster"]
+= Managing {product-long-kafka} on a ROSA cluster
+ifdef::context[:parent-context: {context}]
+:context: deploying-cluster
+
+// Purpose statement for the assembly
+[role="_abstract"]
+
+As an organization administrator, you can deploy {product-long-kafka} within your own controlled environment on your own ROSA cluster.
+
+//Additional line break to resolve mod docs generation error.
+
+[id="proc-installing-on-your-rosa-cluster_{context}"]
+== Installing {product-kafka} on your ROSA cluster
+
+You must install {product-long-kafka} on your {rosa-name} ROSA cluster before users in your organization can create Kafka resources. Use the {product-long-rhoas} `rhoas` command-line interface (CLI) to register your (ROSA) cluster through a call to the Kafka Service Fleet Manager service. This service exposes a Rest API to manage Kafka instances. Registering your ROSA cluster allows the service to install certain necessary add-ons to your cluster so users can provision their Kafka instances.
+
+.Prerequisites
+
+* You have a Red Hat account.
+* You are an organization administrator.
+* You've installed the latest version of the `rhoas` CLI (see {base-url}{installation-guide-url-cli}[Installing and configuring the rhoas CLI^]).
+* You have created a ROSA cluster. {product-long-kafka} uses the bring-your-own-key (BYOK) encryption that you have configured in your ROSA cluster. As a result, volumes used by the service are automatically encrypted using your key. For more information about creating and configuring a ROSA cluster, see https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html[Creating a ROSA cluster with STS using customizations^].
```

Comment:
BYOK is a option. The way it is expressed here it sounds like it is mandatory.    If the user has configured BYOK on their ROSA cluster, the service will use it.  If they haven't the service will just use a platform owned key instead.

Reply:
Thanks @k-wall. I can move it to above section.

Reply:
Looks good now.

Reply:
Thanks @k-wall. If you are happy with the doc, can you give it a LGTM please? I will then merge it and sync the revised ROSA attribute downstream, and create a new PR to capture any QE validation feedback.